RecoVertex/PrimaryVertexProducer/python/OfflinePrimaryVertices_cfi.py

```diff
@@ -11,11 +11,12 @@
     
     TkFilterParameters = cms.PSet(
         algorithm=cms.string('filter'),
-        maxNormalizedChi2 = cms.double(20.0),
+        maxNormalizedChi2 = cms.double(10.0),
         minPixelLayersWithHits=cms.int32(2),
         minSiliconLayersWithHits = cms.int32(5),
-        maxD0Significance = cms.double(5.0), 
+        maxD0Significance = cms.double(4.0), 
         minPt = cms.double(0.0),
+        maxEta = cms.double(2.4),
```

Comment:
On the other hand, very forward tracks have poor dz resolution so it makes sense to exclude them from vertexing. The optimal value may be different of course.